index.html

```diff
@@ -0,0 +1,215 @@
+      <section class="related__wrapper">
+        <article class="related__item item">
+          <a href="#" class="item__link">
+            <h4 class="item__title">Titlu articol sportiv</h4>
+          </a>
+        </article>
+        <article class="related__item item">
+          <a href="#" class="item__link">
+            <h4 class="item__title">
+              Alt titlu articol sportiv pe trei randuri
+            </h4>
+          </a>
+        </article>
+        <article class="related__item item">
+          <a href="#" class="item__link">
+            <h4 class="item__title">
+              Încă un titlu articol sportiv pe trei randuri sau chiar mai mult
+              de 3 randuri
+            </h4>
+          </a>
+        </article>
+      </section>
+      <a href="#" class="related__action">Vezi toate articolele</a>
+    </aside>
+    <footer class="footer">
+      <div class="footer__social social">
+        <ul class="social__wrapper">
+          <li class="social__item">
+            <img
+              src="https://via.placeholder.com/130x35?text=Facebook"
+              alt="Logo of Facebook"
+              width="130"
+              height="35"
+              class="social__image"
+            />
+          </li>
+          <li class="social__item">
+            <img
+              src="https://via.placeholder.com/105x35?text=Twitter"
+              alt="Logo of Twitter"
+              width="105"
+              height="35"
+              class="social__image"
+            />
+          </li>
+        </ul>
+      </div>
+      <div class="footer__nav nav-secondary">
+        <ul class="nav-secondary__wrapper">
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Biciclete </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Piese bicicletă </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Accesorii bicicletă </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Echipamente </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> About </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Blog </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Întrebări frecvente </a>
+          </li>
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Contact </a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer__copyright">
+        <img
+          src="https://via.placeholder.com/100x50?text=Logo"
+          alt="Logo of VeloPlus"
+          width="160"
+          height="50"
+          class="footer__logo"
+        />

```

Comment:
Nu e nevoie sa inchizi tag-ul de imagine, e self closing.
```suggestion
        >
```
```